example/styles.css

```diff
@@ -160,9 +160,6 @@ button:not(.cf-btn) {
 }
 
 .cf-card__footer_message {
```

Comment:
Make sure you make this change to the internal styles in WWW-Style as well @thellimist.